README.md

```diff
@@ -56,13 +57,20 @@ wsk action update myAction myAction.js --kind nodejs:10
 wsk action update myAction myAction.js --kind nodejs:12
 ```
 
+### Node.js v14
+
+```
+wsk action update myAction myAction.js --kind nodejs:14
```

Comment:
This won't work until we actually add the runtime to OpenWhisk but I added it anyway in anticipation of that happening soon.